crates/tests-e2e/src/app.rs

```diff
@@ -25,7 +25,7 @@ pub async fn run_app() -> AppHandle {
         let app_dir = as_string(&app_dir);
         let seed_dir = as_string(&seed_dir);
         tokio::task::spawn_blocking(move || {
-            native::api::run(default_config(), app_dir, seed_dir).expect("Could not run app")
+            native::api::run(default_config(), app_dir, seed_dir, false).expect("Could not run app")
```

Comment:
❓Why is this useful?

Reply:
it stops the annoying long backtrace on failed test.